versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,146 @@
+    --set global.aws.roleName=$ROLE_NAME
+  ```
+
+  :::note
+
+  Monitor the logs for the `rancher-cloud` pod since it is deleted 1 minute after a successful or failed installation.
+
+  ```shell
+  kubectl logs -f rancher-cloud -n cattle-rancher-csp-deployer-system
+  ```
+
+  :::
+
+1. After a successful deployment, running the following command should produce a similar output:
+
+  ```shell
+  kubectl get deployments --all-namespaces
+  ```
+
+  ```shell
+  NAMESPACE                           NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+  cattle-csp-billing-adapter-system   csp-rancher-usage-operator    1/1     1            1           30m
+  cattle-csp-billing-adapter-system   rancher-csp-billing-adapter   1/1     1            1           30m
+  cattle-fleet-local-system           fleet-agent                   1/1     1            1           29m
+  cattle-fleet-system                 fleet-controller              1/1     1            1           29m
+  cattle-fleet-system                 gitjob                        1/1     1            1           29m
+  cattle-provisioning-capi-system     capi-controller-manager       1/1     1            1           28m
+  cattle-system                       rancher                       1/1     1            1           32m
+  cattle-system                       rancher-webhook               1/1     1            1           29m
+  cert-manager                        cert-manager                  1/1     1            1           32m
+  cert-manager                        cert-manager-cainjector       1/1     1            1           32m
+  cert-manager                        cert-manager-webhook          1/1     1            1           32m
+  ingress-nginx                       ingress-nginx-controller      1/1     1            1           33m
+  kube-system                         coredns                       2/2     2            2           38m
+  ```
+
+### Check Helm Chart Installation
+
+Check that the Helm chart installation is complete:
+
+```shell
+helm ls -n cattle-rancher-csp-deployer-system
+```
+
+After completing the Helm chart installation, you can verify the installation was successful:
+
+```shell
+helm status rancher-cloud -n cattle-rancher-csp-deployer-system
+```
+
+Refer to the [Troubleshooting](troubleshooting.md) section for a failed installation.
+
+After the Helm chart installation is complete, Rancher Prime is successfully installed.
+
+## Log into the Rancher Dashboard
+
+You may now log in to the Rancher dashboard by pointing your browser to the Rancher server URL `https://$RANCHER_HOSTNAME`, where the Rancher hostname is the [hostname](#installing-rancher) is your chosen hostname when installing Rancher.
+
+:::note
+
+The Rancher hostname must be resolvable by public DNS. For more details, please refer to the [Prerequisites](prerequisites.md) section.
+
+:::
+
+## Uninstalling Rancher Prime PAYG Offering
```

Comment:
This section is missing the case where only Rancher Prime PAYG is uninstalled, but the EKS cluster is preserved.